skipper.yaml

```diff
@@ -66,4 +66,4 @@ env:
   API_VIP_DNSNAME: $API_VIP_DNSNAME
   DEPLOY_TARGET: $DEPLOY_TARGET
   ASSISTED_SERVICE_HOST: $ASSISTED_SERVICE_HOST
-  LOGS_DEST: $LOGS_DEST
```

Comment:
we need this one. Why to remove?

Reply:
Must have slipped during rebase. Added back